src/lib.rs

```diff
@@ -230,17 +239,50 @@ impl Problem {
             obj_coeffs: vec![],
             var_mins: vec![],
             var_maxs: vec![],
+            var_domains: vec![],
             constraints: vec![],
         }
     }
 
-    /// Add a new variable to the problem.
+    /// Add a new real variable to the problem.
     ///
     /// `obj_coeff` is a coefficient of the term in the objective function corresponding to this
     /// variable, `min` and `max` are the minimum and maximum (inclusive) bounds of this
     /// variable. If one of the bounds is absent, use `f64::NEG_INFINITY` for minimum and
     /// `f64::INFINITY` for maximum.
-    pub fn add_var(&mut self, obj_coeff: f64, (min, max): (f64, f64)) -> Variable {
+    pub fn add_real_var(&mut self, obj_coeff: f64, (min, max): (f64, f64)) -> Variable {
```

Comment:
I'm not sure this is worth introducing a breaking API change. Can we just keep the existing add_var (just updating the documentation), and add the new functions ? This way we don't need to break the existing minilp api.